functions/source/CopyAitDeployment/copy_ait_deployment.py

```diff
@@ -0,0 +1,152 @@
+        download_tar_gz(ait_url, path=dir_path)
+        shutil.copytree(
+            "/tmp/AIT-GUI-2.3.1", "/mnt/efs/ait/AIT-GUI", dirs_exist_ok=True
+        )
+
+        ## AIT DSN
+        # Build directory AIT DSN directory
+        print("Downloading AIT DSN Plugin")
+        pathlib.Path("/mnt/efs/ait/AIT-DSN/").mkdir(parents=True, exist_ok=True)
+        # Download and place into AIT
+        ait_url = "https://github.com/NASA-AMMOS/AIT-DSN/archive/refs/tags/2.0.0.tar.gz"
+        download_tar_gz(ait_url, path=dir_path)
+        shutil.copytree(
+            "/tmp/AIT-DSN-2.0.0", "/mnt/efs/ait/AIT-DSN", dirs_exist_ok=True
+        )
+
+        # Build necessary folders for the AIT DSN plugin
+        pathlib.Path("/mnt/efs/ait/AIT-Core/ait/dsn/cfdp/datasink/outgoing").mkdir(
+            parents=True, exist_ok=True
+        )
+        pathlib.Path("/mnt/efs/ait/AIT-Core/ait/dsn/cfdp/datasink/incoming").mkdir(
+            parents=True, exist_ok=True
+        )
+        pathlib.Path("/mnt/efs/ait/AIT-Core/ait/dsn/cfdp/datasink/tempfiles").mkdir(
+            parents=True, exist_ok=True
+        )
+        pathlib.Path("/mnt/efs/ait/AIT-Core/ait/dsn/cfdp/datasink/pdusink").mkdir(
+            parents=True, exist_ok=True
+        )
+
+        # Download influx dB
+        print("Downloading Influx DB RPM")
+        download_file(
+            url="https://repos.influxdata.com/rhel/6/amd64/stable/influxdb-1.2.4.x86_64.rpm",
+            filename="/tmp/influxdb-1.2.4.x86_64.rpm",
+        )
+        pathlib.Path("/mnt/efs/ait/setup").mkdir(parents=True, exist_ok=True)
+        shutil.copyfile(
+            "/tmp/influxdb-1.2.4.x86_64.rpm",
+            "/mnt/efs/ait/setup/influxdb-1.2.4.x86_64.rpm",
+        )
+
+        ## Configuration files from S3
+        print("Downloading CONFIG files")
+        s3 = boto3.client("s3")
+        bucket = s3.list_objects(Bucket=BUCKET_NAME)
+        for content in bucket["Contents"]:
+            key = content["Key"]
+            filename = f"/tmp/{key}"
+            path, _ = os.path.split(filename)
+            # Make directories
+            pathlib.Path(path).mkdir(parents=True, exist_ok=True)
+            print(f"Downloading {key} into {filename}")
+
+            # Download file into relavent paths
+            s3.download_file(BUCKET_NAME, key, filename)
+
+        # Copy configuration files into mounted EFS
+        shutil.copytree(
+            "/tmp/configs", "/mnt/efs/ait/setup/configs", dirs_exist_ok=True
+        )
+
+        print("All downloads completed...")
+
+    elif event["RequestType"] in ["Delete", "Update"]:
+        # No action needs to be taken for delete or update events
+        status = cfnresponse.SUCCESS
+        responseData["LambaTest"] = "Not Create"
+    else:
+        responseData = {"Message": "Invalid Request Type"}
+
+    path = "/mnt/efs/ait/"
+    print(f"Listing directories in place {path}")
+    print(os.listdir(path))
```

Comment:
Can we replace this with a print of `tree -R`? or something that will give a more complete picture of what's been bootstrapped?

Reply:
looks like:
```
$ tree -R -L 3
.
├── CODEOWNERS
├── LICENSE.txt
├── NOTICE.txt
├── README.md
├── config-bucket
├── configs
│   ├── ait
│   │   ├── ait-config.zip
│   │   ├── cloudwatch-agent-ait.json
│   │   ├── config
│   │   └── httpd_proxy.conf
│   ├── editor
│   │   └── cloudwatch-agent-editor.json
│   └── modules
│       └── openmct-static.tgz
...
```